packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -5,14 +5,43 @@
 import 'basic.dart';
 import 'framework.dart';
 
-/// A place in the overlay that can contain a widget.
+/// A place in an [Overlay] that can contain a widget.
```

Comment:
Yeah, we've been trying to avoid re-using words in the class names even if it makes the docs slightly awkward to read.  At least it gives the reader another chance to guess at what we mean.